### PR TITLE
[OV JS] Extend smart CI for JS API

### PR DESCRIPTION
**Pull request by vishniakov-nikolai**
### Details:
 - Add more validation dependencies

### Tickets:
 - 129334

### CODE REVIEW

**Review by akladiev — APPROVED** [review]

File: .github/components.yml [hunk 1]
```diff
@@ -70,6 +71,7 @@ AUTO_BATCH:
     - TEMPLATE
     - C_API
     - Python_API
+    - JS_API
```
Comment (ilya-lavrenov) [inline .github/components.yml:74]: how HETERO or AUTO BATCH affects JS?
IMO, those all are not required

Comment (ilya-lavrenov) [reply .github/components.yml:74]: maybe you misunderstands how smart CI works.
when AUTO_BATCH is changed, current PR forces to run JS API tests, why do we need it? how AUTO BATCH can break JS API?

Comment (mlukasze) [reply .github/components.yml:74]: as we were instructed adding an area to the list **excludes** i.e JS API from AUTO plugin tests.
will ping you internally about it

Comment (ilya-lavrenov) [reply .github/components.yml:74]: `  revalidate:` is not the same as `exclude` :D

Comment (mlukasze) [reply .github/components.yml:74]: yes, my bad :D 
in that case I'm opting for removing it from this section

File: .github/components.yml [hunk 2]
```diff
@@ -127,15 +125,15 @@ PDPD_FE:
...
+    - MO  
   build:
     - CPU
     - Python_API
     - TOKENIZERS
 
 TFL_FE:
   revalidate:
-    - MO
+    - MO
```
[the remark this answers is not in the document]
Comment (vishniakov-nikolai) [reply .github/components.yml]: Happened accidentally, fixed

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**